core/src/main/java/com/linecorp/armeria/server/Http1RequestDecoder.java

```diff
@@ -335,15 +332,8 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                         // status.
                         final HttpStatusException httpStatusException =
                                 HttpStatusException.of(HttpStatus.REQUEST_ENTITY_TOO_LARGE, cause);
-                        if (!decodedReq.isInitialized()) {
-                            assert decodedReq.needsAggregation();
-                            final StreamingDecodedHttpRequest streamingReq = decodedReq.toAbortedStreaming(
-                                    inboundTrafficController, httpStatusException, shouldReset);
-                            ctx.fireChannelRead(streamingReq);
-                        } else {
-                            decodedReq.setShouldResetOnlyIfRemoteIsOpen(shouldReset);
-                            decodedReq.abortResponse(httpStatusException, true);
-                        }
+                        decodedReq.setShouldResetOnlyIfRemoteIsOpen(shouldReset);
```

Comment:
How about doing this here as well?
https://github.com/line/armeria/blob/0ab172199bdfcbb8473d709b1f563de396910dfa/core/src/main/java/com/linecorp/armeria/server/AbstractHttpResponseHandler.java#L277